docs/en-US/guide/getting-started.md

```diff
@@ -71,7 +71,9 @@ import { createApp } from 'vue'
 import App from './app.vue'
 import { install } from 'vexip-ui'
 
-createApp(App).use(install)
+const app = createApp(App)
+app.use(install)
+app.mount('#app')
```

Comment:
I don't think mostly need to define a variable when creating app, so it's better to directly call the `mount` method.

```suggestion
createApp(App).use(install).mount('#app')
```